src/View.Uno.WinUI/View.Uno.WinUI.csproj

```diff
@@ -1,62 +1,62 @@
-    <PackageReference Include="Uno.WinUI" Version="4.4.13" />
-    <PackageReference Include="Uno.Core" Version="4.0.1" />
-    <PackageReference Include="Uno.SourceGenerationTasks" Version="4.1.0" />
-    <PackageReference Include="System.Reactive" Version="5.0.0" />
-    <PackageReference Include="Uno.UI.Adapter.Microsoft.Extensions.Logging" Version="4.0.7" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="6.0.0" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <!-- Needed for Source Link support -->
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All"/>
-  </ItemGroup>
-	
-  <PropertyGroup Condition="'$(TargetFramework)'=='netstandard2.0'">
-    <DefineConstants>$(DefineConstants);__WASM__</DefineConstants>
-  </PropertyGroup>
-	
-  <ItemGroup Condition="'$(TargetFramework)'=='net6.0-windows10.0.18362'">
-    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.1.3" />
-    <FrameworkReference Update="Microsoft.Windows.SDK.NET.Ref" RuntimeFrameworkVersion="10.0.22000.24" />
-    <FrameworkReference Update="Microsoft.Windows.SDK.NET.Ref" TargetingPackVersion="10.0.22000.24" />
-  </ItemGroup>
-	
-  <ItemGroup Condition="'$(TargetFramework)'!='net6.0-windows10.0.18362'">
-    <Page Include="**\*.xaml" Exclude="bin\**\*.xaml;obj\**\*.xaml" />
-    <Compile Update="**\*.xaml.cs">
-      <DependentUpon>%(Filename)</DependentUpon>
-    </Compile>
-  </ItemGroup>
-	
-  <ItemGroup>
-    <UpToDateCheckInput Include="**\*.xaml" Exclude="bin\**\*.xaml;obj\**\*.xaml" />
-  </ItemGroup>
-	
-  <Import Project="..\View.Uno.Shared\Shared.projitems" Label="Shared" />
+	<PropertyGroup>
+		<TargetFrameworks>net6.0-windows10.0.18362;netstandard2.0;net6.0-ios;net6.0-macos;net6.0-maccatalyst;net6.0-android</TargetFrameworks>
+		<LangVersion>10.0</LangVersion>
+		<!-- Ensures the .xr.xml files are generated in a proper layout folder -->
+		<GenerateLibraryLayout>true</GenerateLibraryLayout>
+		<RootNamespace>Nventive.View</RootNamespace>
+		<Authors>nventive</Authors>
+		<Company>nventive</Company>
+		<AssemblyName>Nventive.View.Uno.WinUI</AssemblyName>
+		<PackageId>Nventive.View.Uno.WinUI</PackageId>
+		<Description>Nventive.View.Uno.WinUI</Description>
+		<GenerateDocumentationFile>true</GenerateDocumentationFile>
+		<PackageTags>view;mvvm;ios;android;maui;winui;WinAppSDK;</PackageTags>
+		<DefineConstants>$(DefineConstants);WINUI</DefineConstants>
+		<!-- TODO: XAML merge will need to be implemented in the future, like it is implemented for the Uno project currently-->
+		<!-- https://github.com/nventive/Nventive.View/issues/48 -->
+
+		<!-- Needed for Source Link support -->
+		<PublishRepositoryUrl>true</PublishRepositoryUrl>
+		<EmbedUntrackedSources>true</EmbedUntrackedSources>
+		<IncludeSymbols>true</IncludeSymbols>
+		<SymbolPackageFormat>snupkg</SymbolPackageFormat>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Uno.WinUI" Version="4.5.14" />
```

Comment:
Keep the lowest minor possible. Updating this forces all uses to use at least that version, which we don't want.